test/test_plugin_helper.rb

```diff
@@ -14,6 +14,8 @@ class ActionController::TestCase
     def add_smart_proxy
       FactoryGirl.create(:smart_proxy, :url => 'http://localhost:8443', :features => [FactoryGirl.create(:feature, :name => 'Openscap')])
       ::ProxyAPI::Features.any_instance.stubs(:features).returns(%w(puppet openscap))
+      versions = { "version" =>"1.11.0", "modules" => { "openscap" => "0.5.3" } }
```

Comment:
We actually don't care much about the Proxy's version. It is this way to formalize the response.
We are only checking ig openscap version is higher than 0.5 (which in turn is dependent on Proxy >= 1.11 on the gemspec...).
will fix the spacing